ocaml/idl/datamodel.ml

```diff
@@ -3442,6 +3442,10 @@ let vdi_resize_online = call
     ~name:"resize_online"
     ~in_oss_since:None
     ~in_product_since:rel_rio
+    ~lifecycle: [
+      Published, rel_rio, "Online resize vdi";
```

Comment:
I think it's fine to have the empty string here:
```
Published, rel_rio, "";
```
Otherwise, I don't know what we could say here, as this was never supported.